features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/EmptyPage.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {Component} from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import {withStyles} from '@material-ui/core/styles';
+import {FormattedMessage} from 'react-intl';
+
+const styles = theme => ({
+    grid:{
+        marginTop:theme.spacing.unit*10
+    },
+    button:{
+        margin:theme.spacing.unit*2
+    },
+    typography:{
+        margin:theme.spacing.unit*2,
+        //height:theme.spacing.unit*4
+    }
+
+});
+
+class EmptyPage extends Component{
+    
+    constructor(props){
```

Comment:
IMHO we could use a functional component for this, since haven't use any lifecycle methods in this component [DOC](https://reactjs.org/docs/components-and-props.html#functional-and-class-components)